### PR TITLE
Implemented Item unit of measure component

### PR DESCRIPTION
**Pull request by mkumbobeaty**
-  [ ] closes I should be able to search for unit of measures #105
-  [ ] closes I should be able to edit unit of measure #106
-  [ ] closes I should be able to refresh list of unit of measures #107
-  [ ] closes I should be able to archive unit of measure #108
-  [ ] closes I should be able to paginating list of unit of measures #103
-  [ ] closes I should be able to create unit of measure  #102 
-  [ ] closes I should be able to view list of unit of measures #101

### CODE REVIEW

File: src/layouts/BaseLayout.js [hunk 1]
```diff
@@ -192,15 +194,19 @@ const breadcrumbNameMap = {
...
   '/resources/utilization': {
     name: 'Utilizations',
     title: 'Resource utilizations',
   },
-  '/resources/warehouses': {
-    name: 'Warehouses',
-    title: 'List of available warehouses',
+  '/resources/itemunit': {
+    name: 'Item Unit',
+    title: 'List of available Item unit of measure',
```
Comment (BenMaruchu) [inline src/layouts/BaseLayout.js]: Can this be `List of available units of measure for items`

File: src/layouts/BaseLayout.js [hunk 2]
```diff
@@ -192,15 +194,19 @@ const breadcrumbNameMap = {
...
   '/resources': { name: 'Resources', title: 'Resource module' },
   '/resources/stock': { name: 'Stock', title: 'List of available stock' },
   '/resources/utilization': {
     name: 'Utilizations',
     title: 'Resource utilizations',
   },
-  '/resources/warehouses': {
-    name: 'Warehouses',
-    title: 'List of available warehouses',
+  '/resources/itemunit': {
```
Comment (BenMaruchu) [inline src/layouts/BaseLayout.js]: I think this should be `unitsofmeasure

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

**Review by navish — CHANGES_REQUESTED** [review]
Update all appearances of functions and variables names from "UnitOfMeasure to ItemUnit" and "ItemUnitOfMeasure to ItemUnit" for consistency
Use Item Unit on component level and Unit of Measure on UI level (what a user sees).

File: src/Resources/components/ItemUnits/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+        if (isEditForm) {
+          const updatedItem = Object.assign({}, itemUnit, values);
+          putItemUnit(
+            updatedItem,
+            () => {
+              notifySuccess('Unit of measure was updated successfully');
+            },
+            () => {
+              notifyError(
+                'Something occurred while updating unit of measure, please try again!'
```
Comment (BenMaruchu) [inline src/Resources/components/ItemUnits/Form/index.js]: This should be `An error ....` and not something. Can you update for all remaining error messages?

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

File: src/Resources/components/ItemUnits/Form/index.js [hunk 4]
```diff
@@ -97,7 +97,7 @@ class ItemUnitForm extends Component {
             },
             () => {
               notifyError(
-                'Something occurred while saving Unit of measure of measure, please try again!'
+                'Something occurred while saving unit of measure of measure, please try again!'
```
Comment (BenMaruchu) [inline src/Resources/components/ItemUnits/Form/index.js]: You forgot this part here

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

File: src/Resources/components/ItemUnits/Form/index.js (hunk 4 above)
Comment (navish) [reply src/Resources/components/ItemUnits/Form/index.js]: Also check for words repetition "of measure of measure"

**Review by BenMaruchu — APPROVED** [review]

**Review by navish — APPROVED** [review]

**Pull request merged**